### PR TITLE
Finished executive summary

### LINKED ISSUE #9: Executive Summary for M1
An executive summary should be evocative. It should capture a reader and make them want to be a part of your idea. In this milestone you will write an executive summary that defines the goals and objectives of your project in language that is easily readable and mental-image evoking. I (or anyone else) should be able to read your executive summary and instantly know a) what you are doing and b) why it is important. Executive summaries should be exciting and interesting. It is the first (and likely the only) chance for you to engage your reader and, in a real world setting, would determine if your product gets funded. An executive summary does not need technical detail to describe interesting functionality. You should mention your product by name without using phrases such as "the class", "the instructor", "project 2" etc.

### PR DESCRIPTION
**Pull request by Vidmaster**
Finished the executive summary and added placeholder sections in the README.md document for additional work. As part of this I also removed all of the project description stuff from Dr. Hale's original document so it didn't get left behind by mistake.

Closes #9

### CODE REVIEW

**Comment by ZexiXin** [comment]
Please merge it into the main branch

File: README.md [hunk 1]
```diff
@@ -1,243 +1,59 @@
...
-
-1. Contribution to the GitHub repo
-1. Activity in the slack channel for your project
-1. Activity on GitHub Projects
-1. In-class participation and participation in team meetings.
-1. Team evaluations
-
-If you do your work and participate, you will get the full team points (**participation_factor**=1), if you don't do anything, you will get 0% of the team points, if you aren't doing enough you will suffer a correspondingly appropriate penalty to your grade. This means that you can **FAIL** even if your team succeeds - so don't sluff off and expect the team to carry you.
+|Some resource| No | Bob | Some description  |
+|e.g. PLC unit | Yes | Gary | A programmable logic controller from Siemens for investigation.|
```
Comment (ZexiXin) [inline README.md:55]: Don't we need to list the Gantt tool at here? I just wondered

**Review by ZexiXin — APPROVED** [review]
Beside what I have added on Resources Needed section. everything is good

File: README.md (hunk 1 above)
Comment (Vidmaster) [reply README.md:55]: I agree! This was just the placeholder from Dr. Hale's version which I copied purely for the formatting though, so I'll leave that for Bryan to add when he does that section.

**Pull request merged**